.dockerignore

```diff
@@ -0,0 +1,15 @@
+__pycache__
+*.pyc
+*.pyo
+*.pyd
+*.db
+*.env
+README.md
```

Comment:
Consider whether you need to include README.md in the Docker image. If it's not required for the application to run, it's good to keep it in .dockerignore.